### ISSUE TITLE
Regression: preloaded container images are ignored

### ISSUE THREAD

**Issue by dabooz** [open-horizon/anax#649]
At some point in the recent past, anax's support for using preloaded images was removed. I think it might have been when support was added to pull images based on the deployment configuration in service and workload definitions. In any case, anax should use images that are already loaded in the host's local docker repo and avoid the network overhead and time required to pull the image every time it is needed.

**Comment by linggao** [comment]
I went through the code.  The logic is this:
1. There are 2 types of image fetching. The first one is fetching it from a docker repository. The second is fetching the image files from an image server and then call docker.imageloade to load it.
2. The first one does not use the skipCheck function because the docker will do it for you. If you  do a `docker pull` from commend line for a image that is already loaded, the docker  will not re-fetch the image. I also tested it for "dockerclient" that we are using in anax, it has the same behaviour.
3. The second case uses the skipCheck function. If the image is loaded, the image file will not be fetched from the image server.
